ds_judgements_public_ui/sass/includes/_judgment_text_toolbar.scss

```diff
@@ -76,10 +76,15 @@
     margin-top: $space-2;
     margin-bottom: $space-4;
 
-    font-size: $typography-xl-text-size;
+    font-size: 1.8rem;
     color: $color-almost-black;
     text-align: center;
     text-wrap: balance;
+
+    @media (max-width: $grid-breakpoint-medium) {
+      margin-top: $space-4;
+      font-size: 1.6rem;
```

Comment:
Same here about size variables.

Reply:
Same here WRT font sizes @Terry-Price what do you think?

Reply:
@Terry-Price please update us then I will change accordingly as this was specific in style guide

Reply:
if i use  ` $typography-lg-text-size` which is 1.3rem instead @Terry-Price @jlhdxw ?
![image](https://github.com/user-attachments/assets/e65f0e36-9508-4f39-86dc-48a6e677bb34)